src/app/functions/l10n/cronjobs.ts

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { dirname, resolve } from "path";
+import { fileURLToPath } from "url";
+import { readFileSync, readdirSync } from "fs";
+import { MarkupParser, ReactLocalization } from "@fluent/react";
+import { JSDOM } from "jsdom";
+import {
+  GetL10n,
+  createGetL10n,
+  createGetL10nBundles,
+  type ExtendedReactLocalization,
+  type GetL10nBundles,
+} from ".";
+import type { SanitizedSubscriberRow } from "../server/sanitize";
+
+export function getEmailL10n(
+  subscriber: SanitizedSubscriberRow,
+): ExtendedReactLocalization {
+  // We don't have a runtime language when we email people, so use their
+  // language setting from when they signed up for their Mozilla account:
+  return getL10n(subscriber.signup_language ?? "en");
+}
+
+export type LocaleData = {
+  locale: string;
+  bundleSources: string[];
+};
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const ftlRoot = resolve(__dirname, `../../locales/`);
+export const getL10nBundles: GetL10nBundles = createGetL10nBundles({
+  availableLocales: readdirSync(ftlRoot),
+  // TODO: Make this optional in `createGetL10nBundles`, which would then make
+  //       it required in the newly-created function:
+  getAcceptLangHeader: () => "en",
+  loadLocaleFiles: (locale) => {
+    const referenceStringsPath = resolve(__dirname, `../../locales/${locale}/`);
+    const ftlPaths = readdirSync(referenceStringsPath).map((filename) =>
+      resolve(referenceStringsPath, filename),
+    );
+
+    return ftlPaths.map((filePath) => readFileSync(filePath, "utf-8"));
+  },
+  loadPendingStrings: () => {
+    const pendingStringsPath = resolve(__dirname, "../../locales-pending/");
+    const ftlPaths = readdirSync(pendingStringsPath).map((filename) =>
+      resolve(pendingStringsPath, filename),
+    );
+
+    return ftlPaths.map((filePath) => readFileSync(filePath, "utf-8"));
+  },
+});
+
+const parseMarkup: MarkupParser = (str) => {
+  if (!str.includes("<") && !str.includes(">")) {
```

Comment:
Heh good point, added a comment in 4faa8fdf1.

It might be a case of premature optimisation, but since the vast majority of strings don't include HTML tags, I figured it might be good to avoid initialising JSDOM for the strings that don't.